src/features/content-explorer/content-explorer/__tests__/ContentExplorerActionButtons.test.js

```diff
@@ -324,16 +324,61 @@ describe('features/content-explorer/content-explorer/ContentExplorerActionButton
     });
 
     describe('renderStatus', () => {
-        test('should show status message for multi select', () => {
-            const item = { id: '0', name: 'item1' };
-            const selectedItems = { '0': item };
+        let item;
+        let selectedItems;
+
+        beforeEach(() => {
+            item = { id: '0', name: 'item1' };
+            selectedItems = { '0': item };
+        });
+
+        test('should show default status message for multi select', () => {
             const wrapper = renderComponent({
                 contentExplorerMode: ContentExplorerModes.MULTI_SELECT,
                 selectedItems,
             });
+            const statusMessageId = wrapper
+                .find('.status-message')
+                .find('FormattedMessage')
+                .prop('id');
 
-            expect(wrapper.find('.status-message').length).toEqual(1);
+            expect(statusMessageId).toEqual('boxui.contentExplorer.numSelected');
+        });
+
+        test('should show status message for multi select when items are selected and we can include subfolders and can select all', () => {
+            const canIncludeSubfolders = true;
+            const isSelectAllAllowed = true;
+            const wrapper = renderComponent({
+                contentExplorerMode: ContentExplorerModes.MULTI_SELECT,
+                selectedItems,
+                canIncludeSubfolders,
+                isSelectAllAllowed,
+            });
```

Comment:
you can inline the values for `canIncludeSubfolders` and `isSelectAllAllowed` directly  in the object

```
const wrapper = renderComponent({
    contentExplorerMode: ContentExplorerModes.MULTI_SELECT,
    selectedItems,
    canIncludeSubfolders: true,
    isSelectAllAllowed: true,
});
```
as they're not providing much value as variables

Reply:
same for the bottom test